OJP/OJP_Trips.xsd

```diff
@@ -210,6 +258,22 @@
 			</xs:choice>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:simpleType name="AccessibilityPreferenceEnumeration">
+		<xs:annotation>
+			<xs:documentation>Allowed values for a AccessibilityPreference.</xs:documentation>
```

Comment:
only when routing relevant

Reply:
I think this is the most crucial point: "The expectation of the backend must be in good detail in the documentation."